openbb_sdk/openbb/package/crypto.py

```diff
@@ -22,54 +21,43 @@ def __repr__(self) -> str:
     @validate_arguments
     def load(
         self,
-        symbol: typing_extensions.Annotated[
-            Union[str, List[str]],
-            OpenBBCustomParameter(description="Symbol to get data for."),
```

Comment:
@hjoaquim do we need to keep `OpenBBCustomParameter` ?

Reply:
@montezdesousa We use it so that argparse can work with the Platform on the Terminal.

Reply:
I'm aware, is there any good alternative @IgorWounds @hjoaquim? Otherwise we need to keep this @tehcoderer

Reply:
yeah I took it out init cause it didn't work with basemodel, just pushed an update that uses `dataclass` instead and works